Project.toml

```diff
@@ -3,41 +3,44 @@ uuid = "4d6a76a9-bfbc-5492-8924-cf6ed7875f06"
 license = "ISC"
 description = "Econometrics in Julia"
 maintainer = "José Bayoán Santiago Calderón"
-version = "0.2.8"
+version = "0.2.9"
 
 [deps]
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 CategoricalArrays = "324d7699-5711-5eae-9e2f-1d82baa6b597"
 CompilerSupportLibraries_jll = "e66e0078-7015-5450-92f7-15fbd957f2ae"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 JLLWrappers = "692b3bcd-3c85-4b1f-b108-f13ce0eb3210"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MLJBase = "a7f614a8-145f-11e9-1d2a-a57a1082229d"
 MLJModelInterface = "e80e1ace-859a-464e-9ed9-23947d8ae3ea"
 Optim = "429524aa-4258-5aef-a3af-852621145aeb"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+RDatasets = "ce6b1742-4840-55fa-b093-852dadbb1d8b"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 StatsModels = "3eaba693-59b7-5ba5-a881-562e759f1c8d"
 TableOperations = "ab02a1b2-a7df-11e8-156e-fb1833f50b87"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
-CategoricalArrays = "0.8, 0.9"
-Distributions = "0.23, 0.24"
+CategoricalArrays = "0.8, 0.9, 0.10"
+Distributions = "0.23, 0.24, 0.25"
 FillArrays = "0.9, 0.10, 0.11"
 ForwardDiff = "0.10"
-MLJBase = "0.15, 0.16"
-MLJModelInterface = "0.3"
+JLLWrappers = "1"
```

Comment:
This must be the first time I see a compat with `JLLWrappers` outside of a JLL package.

Reply:
CompatHelper gets mad at me if I don't I guess. Guess it should work the same as any other package but probably not vital.